custom_components/ha_kia_hyundai/select.py

```diff
@@ -84,6 +84,7 @@ async def async_setup_entry(
     async_add_entities(
         SeatSelect(coordinator, select_description)
         for select_description in SEAT_SELECTIONS
+        if coordinator.has_climate_seats
```

Comment:
This will check if seats are installed, if not it will skip calling the exists_fn and adding the entity. Same logic in select.py

Reply:
I guess I’m assuming that all vehicles with ventilated seats have heated seats as well, but I suppose that might not be the case. I’ll redo the check in the coordinator to account for both.

Reply:
After looking at my vehicle that has heated but not cooled seats, it shows 1 for both options, so I'm not sure if it matters but I added a check for both.

`'vehicleFeature': {'remoteFeature': {'lock': '1', 'unlock': '1', 'start': '3', 'stop': '1', 'scheduleCount': '2', 'inVehicleSchedule': '1', 'heatedSteeringWheel': '0', 'heatedSideMirror': '1', 'heatedRearWindow': '1', 'heatedSeat': '1', 'ventSeat': '1'`